pkg/cloud/metadata/ecs.go

```diff
@@ -23,6 +23,7 @@ type InstanceIdentityDocument struct {
 	InstanceID   string `json:"instance-id"`
 	InstanceType string `json:"instance-type"`
 	SerialNumber string `json:"serial-number"`
+	AliUID       string `json:"owner-account-id"`
```

Comment:
```suggestion
	OwnerAccountID string `json:"owner-account-id"`
```
It can be confusing if the json tag is differ from field name